modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/cheems.dm

```diff
@@ -0,0 +1,31 @@
+/mob/living/simple_animal/pet/dog/cheems
+	name = "\proper Cheems"
+	real_name = "Cheems"
+	gender = MALE
+	desc = "It's the Cargo's overfed, yet still beloved dog."
+	icon = 'modular_skyrat/master_files/icons/mob/pets.dmi'
+	icon_state = "cheems"
+	icon_dead = "cheems_dead"
+	icon_living = "cheems"
+	speak = list("Borf!", "Boof!", "Bork!", "Bowwow!", "Burbger...", "Pizza cramte.", "No horny.")
```

Comment:
```suggestion
	speak = list("Borf!", "Boof!", "Bork!", "Bowwow!")
```

Reply:
let us keep AT LEAST one of the funny lines